go-worker/main.go

```diff
@@ -27,11 +30,23 @@ func main() {
 		panic(fmt.Errorf("Couldnt connect to redis"))
 	}
 	redisCache := cache.NewRedisCache(redisClient)
+	pubsub := pubsub.NewRedisPubSub(redisClient)
 
 	baseClient := client.NewClient(&http.Client{Timeout: time.Second * 10})
-	footballAPIClient := client.NewFootballAPIClient(baseClient, "https://api-football-v1.p.rapidapi.com/v3/fixtures")
-	matchService := service.NewMatchService(redisCache, footballAPIClient)
+	footballAPIClient := client.NewFootballAPIClient(baseClient, os.Getenv(constants.FootballApiUrl))
```

Comment:
since passing from main the url, also pass the api key to stay consistant and only access env var from main